system/upgrades_table.sql

```diff
@@ -0,0 +1,17 @@
+
+-- adds informations table used by the new db_manager
```

Comment:
"new" will soon no longer be outdated ;)

Reply:
and db_manager should be pum ;)

Reply:
Fixed!